docs/How  to use the epos plugin.md

```diff
@@ -0,0 +1,136 @@
+# How to use the Epos plugin of the FAIR evaluator 
+To use the fair evalater you need access to a terminal 
+
+## 1.Follow the steps indicated in the  documentation
+https://github.com/EOSC-synergy/FAIR_eva/blob/main/docs/index.md which are executting the following commands
+
+git clone https://github.com/EOSC-synergy/FAIR_eva.git
+cd ./FAIR_eva
+pip3 install -r requirements.txt
+cp config.ini.template config.ini
+
+Basically clone the github repo in your computer, change into de adecuate folder, install all the python modules necesary for the program  and copy the template for the configuration file 
+
+## 2.Run fair.py (python3 fair.py)
+
+Now you have the API running in that terminal (let's call it terminal 1) here you can see all the steps the API is following.
+
+
+## 3.Make a request to evaluate  
+On another terminal (terminal 2) you will make a curl request to that API. Here we will get the responses
+In order to do that we use :
+curl  -H  "accept: application/json"\
+      -H  "Content-Type: application/json" \
+      -d '{"id":"7c9dfb3c-7db0-4424-8843-ada2143b00a0","lang":"es","oai_base":  "https://ics-c.epos-ip.org/development/k8s-epos-deploy/dt-geo/api/v1","repo":"epos"}'\
+      -X POST "http://localhost:9090/v1.0/rda/rda_all"
+
+It will return an evaluation of all the tests 
+## DONE!
+
+That one is a particular example there are two things you can change:
+1.- The id to the one you are looking for 
+2.- The end of the -X POST "http://localhost:9090/v1.0/rda/rda_all" you can change rda all for a code to another test.
+
+A simple way to calculate
+If you are only interested in the scores and how your item qualifies in each category you can use fair-eva.py a simple script that writes the points in each category.
+This script is located in the scripts folder.
+To use it you simply need to execute the command (you still need the terminal 1 open with the API running):
+
+python3 scripts/fair-eva.py -i 7c9dfb3c-7db0-4424-8843-ada2143b00a0 -p epos -r https://ics-c.epos-ip.org/development/k8s-epos-deploy/dt-geo/api/v1 
+
+Where in the -i you put the UUID to evaluate, in -p the plugin  and in -r the endpoint. The ones in the example are from epos. 
```

Comment:
Add a caption of `--help` and remove this text

Reply:
done